package.json

```diff
@@ -1,6 +1,14 @@
 {
   "name": "@fractal-framework/fractal-contracts",
-  "version": "0.1.0",
+  "version": "1.0.0",
+  "files": [
```

Comment:
Do we want to include the deployments folder here for publishing the deployed contract addresses?

Reply:
I am not sure? Do we -- we did not include this in the previous package

Reply:
I think that's what @herbig asked for